tedana/resources/decision_trees/minimal.json

```diff
@@ -132,7 +147,7 @@
                 "subset_decide_comps": "unclassified",
                 "rho_elbow_type": "liberal",
                 "log_extra_info": "",
-                "log_extra_report": ""
+                "log_extra_report": "The 'liberal' rho elbow is calculated from all (unclassified?) components, for later steps."
```

Comment:
I need to look at the rho elbow function to figure out how both unclassified and all components are used.

Reply:
Honestly, the elbow logic doesn't make sense to me, but the end result seems to be that this step calculates the elbow as the maximum of two values: the elbow calculated from all components and elbow calculated from "unclassified" components at this point in the decision tree.

Reply:
This is one place worth consideration. I really don't like the rho elbow calculation in the kundu tree which seems to skew low (i.e. more component are skewed towards reject). For the minimal tree, I used 2 of the 3 same calculations, but skewed it to be as high as possible. Since the minimal tree rejects nearly everything that is greater than the rho elbow threshold, I wanted the tree to be conservative and not reject edge cases.

I'll need to relook to confirm, but I think the elbow is calculated on all components and on just the remaining unclassified components. The used threshold is the greater value of the two.